dev-tools/ecs-migration.yml

```diff
@@ -891,6 +891,10 @@
   to: network.forwarded_ip
   alias: false
 
+## Redis
+- from: redis.error
+  to: error.message
+  alias: true
```

Comment:
Same minor heads up as for [Thrift error messages](https://github.com/elastic/beats/pull/10125#pullrequestreview-193736784). Ingest Node suggests outputting IN errors in `error.message`. If Pb doesn't use IN at all, I think we're good.

What you're doing here is the correct usage for `error.message`, IN example is what will have to change.

Reply:
I'm going to undo the `error.message` change I did here. It doesn't align well with the other protocols and makes the value of the field confusing because you are not sure if it's a protocol error (bad SQL query) or a Packetbeat error (packet loss detected).